### DISCUSSION TITLE
vcpkg.json

### DISCUSSION THREAD

**Discussion by idzard-intentdev** [IntentDev/touchpy#53] [General]
on github main vcpkg.json has no dependencies

```
{
  "dependencies": [
  ]
}
```

but shouldn't it contain this?

```
{
	"dependencies": [
	  "vulkan-memory-allocator",
	  "gtest",
	  {
		"name": "spdlog",
		"version>=": "1.13.0"
	  }
	]
  }
```

**Comment by keithlostracco** [comment]
Yep I removed them for now because it slows down the cmake build time
considerably. If and when more people work on the project we can add it
back but it's much faster for cmake config and generate if the packages are
installed globally.
